### ISSUE TITLE
Fix Ensembl transcript mapping in VF/LOVD

### ISSUE THREAD

**Issue by PeteCausey-Freeman** [openvar/variantValidator#385]
**Fix Ensembl transcripts in VF**
See https://github.com/openvar/variantValidator/issues/204

**Describe the solution you'd like**
Currently, the correct mappers are not being selected for Ensembl aligners

**Comment by Peter-J-Freeman** [comment]
@al-baba . We will plan this one together on here. Let me know when you are ready for some instructions

**Comment by al-baba** [comment]
Thanks @Peter-J-Freeman, awaiting instructions.

**Comment by Peter-J-Freeman** [comment]
Fantastic. What machine are you running on? We need to get VV installed. Instructions here. https://github.com/openvar/variantValidator#readme. Before you start let me know the machine and I will advise. Needs to be LINUX so mac or a VM on windows

**Comment by al-baba** [comment]
Still have the VM on windows as per the PGcert: so Linux Mint (I think 20.3 - just away from the machine for the weekend so can't check which one exactly).

**Comment by Peter-J-Freeman** [comment]
Fab. We will need to increase the capacity of your VM. Might be worth doing this on a call. Email me so we can set up

**Comment by al-baba** [comment]
Hi @Peter-J-Freeman, I'm having trouble at creating a conda (miniconda3) environment with the YAML file provided. 


Must it be Anaconda and not miniconda? 

<pre><font color="#8AE234"><b>alexander@alexander-VirtualBox</b></font>:<font color="#729FCF"><b>~/variantValidator</b></font>$ conda env create --file environment.yml
Collecting package metadata (repodata.json): / <font color="#8AE234"><b>alexander@alexander-VirtualBox</b></font>:<font color="#729FCF"><b>~/variantValidator</b></font>$</pre>

**Comment by Peter-J-Freeman** [comment]
It is more likely a config issue with your miniconda. Both Anaconda and miniconda are interchangeable in this instance. 

We will set up a call

**Comment by al-baba** [comment]
![Screenshot (12)](https://user-images.githubusercontent.com/57296226/179736574-8211ab72-fde5-4def-8da0-be4f4f7efc4f.png)
@Peter-J-Freeman 
It was there in front of us! 'Tools!'

I've increased to 40GB and will give it a go. 

The base memory increase we did only served to crash the VM. Do I need to purchase and expand my PC's RAM?

**Comment by Peter-J-Freeman** [comment]
Nah, just reduce this a bit. How much can you allocate? Thanks for the image. I may steal it for teaching

**Comment by al-baba** [comment]
Cant seem to reduce down from 40GB now that I've raised it. 

I have had to bring the Base Memory back down to 1832MB to keep it working.

I'm on a acer revo rn96 mini-PC, i3 with 4GB RAM (3.74 GB usable). 

By all means use the image! :-)

**Comment by al-baba** [comment]
@Peter-J-Freeman 

VV installed !

![image](https://user-images.githubusercontent.com/57296226/196525939-2fbb9948-0f4d-4cd1-bc6a-6442a963cc85.png)
